crates/proof-of-sql/examples/hello_world/README.md

```diff
@@ -12,7 +12,7 @@ This example demonstrates generating and verifying a proof of the query `SELECT
 #### Run
 
 ```bash
-cargo run --example hello_world
+cargo run --example hello_world --features="test"
```

Comment:
@Abiji-2020 You are on the right track maybe you can also add a note like this 
```suggestion
:::note
To run this example with the `blitzar` feature:
cargo run --example hello_world --features="blitzar test
```

Reply:
Actually I had a doubt on adding this as it was not required  in the examples, hence I had asked this in the issue part. So we need to include it right?

Reply:
Yes, according to this task:
> Update the READMEs to add examples of how to run this with and without the blitzar feature enabled.

We need to update with and without `blitzar' feature flag

Reply:
Ok updated them in the following commits

Reply:
`blitzar` is a default feature. So, we need the --no-default-features enabled to turn it off.

Reply:
Ohh okay will update it in both the `readme.md` and `CI`